src/test/java/FileUtilityTest.java

```diff
@@ -1,31 +1,49 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Assumptions;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 import utilities.FileUtility;
 import utilities.I18NUtility;
+import utilities.OS;
 import utilities.PropertyUtility;
 import utilities.ShellUtility;
+
```

Comment:
```suggestion

```

Reply:
Will be addressed in a future task for linting.